tests/test_env.py

```diff
@@ -83,8 +83,11 @@ def test_env_accessors(gdalenv):
     setenv(foo='1', bar='2')
     expected = default_options.copy()
     expected.update({'foo': '1', 'bar': '2'})
-    assert getenv() == rasterio.env.local._env.options
-    assert getenv() == expected
+    items = getenv()
+    assert items == rasterio.env.local._env.options
+    # Comparison below requires removal of GDAL_DATA.
+    items.pop('GDAL_DATA', None)
+    assert items == expected
```

Comment:
`GDAL_DATA` is a config item now and popping it off lets us make the comparisons without any other modifications.